### DISCUSSION TITLE
Fastest way to download MIMIC-CXR-JPG onto local machine?

### DISCUSSION THREAD

**Discussion by rthapa84** [MIT-LCP/mimic-code#1725] [MIMIC-IV-CXR] [answered]
What is the fastest way to download the CXR data onto my local computer? Right now, my download speed is showing 500KB/sec, and it does not seem to be the issue from my network. Seems like there is some kind of cap on download speed. 

Also, the clinical notes seems to be present in MIMIC-CXR, but not in MIMIC-CXR-JPG. I am just trying to download MIMIC-CXR-JPG. How can I get access to those notes, without having to download the whole original MIMIC-CXR?

**Comment by tompollard** [comment, marked as answer]
> What is the fastest way to download the CXR data onto my local computer? Right now, my download speed is showing 500KB/sec, and it does not seem to be the issue from my network. Seems like there is some kind of cap on download speed.

The fastest way is to download the data from Google Cloud. You should see an option in the "Files" section that points to the Google bucket (and in some cases suggests a gsutil command for downloading the data). Note that you may have to provide a Google Cloud ID to cover egress costs.

> Seems like there is some kind of cap on download speed.

We're at full bandwidth on downloads from MIT servers unfortunately. To help resolve this issue, we are working on cloud integration and are in the process of moving data onto AWS Cloud too (which will not incur egress costs).

**Reply by rthapa84** [reply]
I see. Thank makes sense. Thank you for you response. One more question:

The clinical notes seems to be present in MIMIC-CXR, but not in MIMIC-CXR-JPG. I am just trying to download MIMIC-CXR-JPG. How can I get access to those notes, without having to download the whole original MIMIC-CXR?

**Reply by tompollard** [reply]
Please could you try to keep issues and discussions focused on a single issue? It helps to keep conversations clear, discovery of relevant discussions, etc.

> How can I get access to those notes, without having to download the whole original MIMIC-CXR?

You can download the reports file from MIMIC-CXR without downloading the whole dataset. Getting the reports from Google Cloud bucket would be quickest (~100MB, so shouldn't take too long).

**Reply by rthapa84** [reply]
Good point. My apologies. Will keep that in mind for future. Thanks!

**Reply by swhoosh** [reply]
<img width="903" alt="image" src="https://github.com/MIT-LCP/mimic-code/assets/70953480/4a5b1bf6-237d-4b3e-aa4b-92c2dfaccc03">

Hi. Apparently I couldn't find any Google Cloud bucket or any other methods to download this dataset. Was it just removed? Are there any other ways for me to download this quickly? I tried using parallel but got `[connection failed](failed: Cannot assign requested address.)` maybe due to bandwidth cap.

**Reply by tompollard** [reply]
@swhoosh It looks like https://physionet.org/content/mimic-cxr-jpg/2.0.0/ is available on Google Cloud, but https://physionet.org/content/mimic-cxr-jpg/2.1.0/ hasn't yet been transferred. We will sync the newer version soon, but in the meantime you could consider using 2.0.0 (see release notes section for changes between versions).

**Reply by swhoosh** [reply]
Found it. Thanks you so much! Btw do you know the estimated cost of downloading via Google Cloud? Is it 0.12 USD per GB? I'm not sure about the pricing.

**Reply by tompollard** [reply]
https://cloud.google.com/storage/pricing#network-egress

**Reply by amitsaha** [reply]
Hi @swhoosh did you find a cost-effective way to get access to the data from google cloud?

**Comment by junxiant** [comment]
Hello, has there been any updates on the fastest way without going to google cloud that will require paying?

**Comment by amitsaha** [comment]
If you are a PhD student, you can try for
https://edu.google.com/intl/ALL_us/programs/credits/research/?modal_active=none
to get credits for Google cloud

**Reply by adhiiisetiawan** [reply]
Hi @amitsaha is it possible for download using new account of google cloud? if i'm not mistaken, new account will got 300 USD free credit

**Comment by issayibrahim** [comment]
can anyone hlp me get the dataset?

**Reply by alistairewj** [reply]
The dataset is now hosted on AWS via the Open Data program. If you navigate to the project page and scroll to just above the files section, there is a button to request access via AWS. There are further instructions on that page. This requires an AWS account, but it is free to the user.

**Comment by issayibrahim** [comment]
Thanks for the info,
I'm in Syria and can't create accounts on AWS or any global corporate
platforms.

Could you please help me get the dataset through a direct link, torrent, or
any other mirror that doesn't require registration?

I'd really appreciate your help.

**Reply by alistairewj** [reply]
The only sources are AWS and PhysioNet. Given your circumstances, downloading from PhysioNet is the best option.
